src/ol/source/tilevectorsource.js

```diff
@@ -281,6 +299,88 @@ ol.source.TileVector.prototype.loadFeatures =
 
 
 /**
+ * @param {goog.Uri|string} url URL.
+ * @param {ol.proj.Projection} projection Projection.
+ * @param {function(this: T, Array.<ol.Feature>)} success Success Callback.
+ * @param {T} thisArg Value to use as `this` when executing `success` or
+ *     `error`.
+ * @template T
+ * @private
+ */
+ol.source.TileVector.prototype.loadFeaturesFromURL_ =
```

Comment:
Yeah, I even added a fixme to tilevectorsource.js to mention that problem. I haven't been able to find a good way to share that code, but I'll try harder. I did not want to spend too much time because the "tile vector" code is going to change in the future.